60-steam-input.rules

```diff
@@ -112,3 +112,12 @@ KERNEL=="hidraw*", KERNELS=="*0955:7214*", MODE="0660", TAG+="uaccess"
 
 # Astro C40
 KERNEL=="hidraw*", ATTRS{idVendor}=="9886", ATTRS{idProduct}=="0025", MODE="0660", TAG+="uaccess"
+
+# Saitek Saitek Pro Flight Yoke
+KERNEL=="hidraw*", ATTRS{idVendor}=="06a3", ATTRS{idProduct}=="0bac", MODE="0660", TAG+="uaccess"
```

Comment:
I'm told this should be unnecessary, because Steam and/or Proton only use raw HID for Xbox/Playstation-style gamepads, not for other game controllers like flight sticks and steering wheels (for which games normally use the evdev API through the `/dev/input/event*` device nodes). But I might be wrong about that? Or a particular game might be going behind their back and using raw HID?

Do you have a concrete example of a pattern for using these controllers that does not work without this rule, and does work with this rule? Some relevant information:

* what game?
* native Linux game, or Proton?
* what Proton version, if any?
* what Steam Input settings?